R/create_map_measurements.R

```diff
@@ -0,0 +1,24 @@
+#' Create map
+#'
+#' Fetch a map of Groningen from OpenStreetMap
+#' and save it as a PNG-file.
+#'
+#' @param png_filename name of the PNG that the map will be saved to
+#' @export
+create_map_measurements <- function(png_filename, show_measurements=TRUE) {
```

Comment:
I feel `create_map_measurements` is a bit vague and `create_map_with_car_traffic_densities` to be the more descriptive name. I suggest this function to be renamed to a more descriptive name.